src/ctc/cli/commands/data/events_command.py

```diff
@@ -104,15 +114,20 @@ async def async_events_command(
                 ],
             )
             events.index.name = 'block'
-            events = events[
-                [
-                    column
-                    for column in events.columns
-                    if column.startswith('arg__')
-                ]
+            columns = [
+                column
+                for column in events.columns
+                if column.startswith('arg__')
             ]
+            if timestamps:
+                columns.insert(0, 'timestamp')
+                events = events.astype({'timestamp': 'str'})
```

Comment:
this snipped that inserts the field should be moved to evm.event_utils.event_crud.async_get_events instead of here

that way it can be used not just by the cli interface but also all the python functions

Reply:
@sslivkoff I added the snipped into `evm.event_utils.event_crud.async_get_events` but I also need something here because the code only display the index + columns starting with `arg__`. is it ok with you?